cmd/policy/policy.go

```diff
@@ -83,7 +83,7 @@ func NewCommand(config *settings.Config, preRunE validator.Validator) *cobra.Com
 				return nil
 			},
 			Args:    cobra.ExactArgs(1),
-			Example: `policy push ./policy_bundle_dir_path --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f --context config`,
+			Example: `policy push ./config --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f`,
```

Comment:
The folder to be pushed being called config is confusing. 

Policy push ./policies seems better.

There's the confusion between the context config and the folder config...